### ISSUE TITLE
Logging level is bot configurable

### ISSUE THREAD

**Issue by weiser** [seomoz/qless-js#41]
I am unable to configure the level at which I want QlessClient to log messages.

Given the following configuration of a worker:
```
const QLESS_CONFIG = {
  clientConfig: { url: process.env.REDIS_URL },
  interval: 60 * 1000,
  logLevel: 'debug', // 
  queueNames: ['q1', 'q2'],
};

const worker = new qless.workers.Multi(QLESS_CONFIG);
worker.run();
```

I'd expect to see qless' internal logger to print all `debug` statements.

However, it appears that the logger is hardcoded to only print [`warn` or higher](https://github.com/seomoz/qless-js/blob/master/lib/logger.js#L18)

I'm happy to make a PR to address this.

**Comment by dlecocq** [comment]
Does this approach not work? https://github.com/seomoz/qless-js/blob/f45df9da74eebc1a48f87ea3e5294f68f5640240/bin/qless-js-worker.js#L32

**Comment by weiser** [comment]
Thanks for pointing that out.  FWIW, the context of this is running a qless worker within an integration test.

I think my source of confusion is that I'd expect to be able to pass the logging level to the worker, like:

```
const QLESS_CONFIG = {
  clientConfig: { url: process.env.REDIS_URL },
  interval: 60 * 1000,
  logLevel: 'debug', // 
  queueNames: ['q1', 'q2'],
};

const worker = new qless.workers.Multi(QLESS_CONFIG); //the worker logs all things above `debug`, since that is what `logLevel` is.
worker.run();
```

and that `logLevel` would determine what level the logger logs.  Whereas it seems that the logger is configured separately from the worker.

So, it is a bit of a surprise, but running:
```
...
qless.logger.level = 'debug'
const worker = new qless.workers.Multi(QLESS_CONFIG);
...
```

does the needful.

I'm not sure if it worth it to keep this issue open.  On one hand, if folk are using qless inside integration tests and they want qless to be loud (in our case, travis builds fail infrequently with a `QlessClient {}` error) it'd be good to know about this.

However, if the main use case is to run a worker through one of the `./bin/*.js` scripts, it doesn't seem super helpful to keep this issue open.

Thoughts?

**Comment by dlecocq** [comment]
Yeah, I was wondering how you were invoking it, but that seems like a legitimate use case.

I'd be open to a PR for this, but I don't see it as the most pressing thing in the world. I think the reason it's not part of the worker creation config is because it's not the case that each worker has a logger - it's global to the library.

**Comment by weiser** [comment]
> I think the reason it's not part of the worker creation config is because it's not the case that each worker has a logger - it's global to the library.

Yeah, that's reasonable.

What if I made a PR against the docs that shows how to create a worker, and configure the log level of qless, that could be used in, for example, testing frameworks?

**Comment by dlecocq** [comment]
Sounds great! Sorry you had to dig around and ultimately open an issue to get it sorted :-/
